src/pages/guides/how-google-fixes-vulnerabilities.tsx

```diff
@@ -0,0 +1,48 @@
+import { FunctionComponent } from 'react'
+
+import { Layout, GatedResourceLayout } from '@components'
+
+export const Guide: FunctionComponent = () => (
+    <Layout
+        meta={{
+            title: 'How Google and Others and Fix Vulnerabilities | Sourcegraph',
```

Comment:
I'm thinking this should be "*find* and fix" and also be read without capitalization on "Others", "Find", "Fix", and "Vulnerabilities".

```suggestion
            title: 'How Google and others find and fix vulnerabilities | Sourcegraph',
```

@fabicastp can you shed some light on the capitalization here being more of a sentence-like title?

Reply:
@bretthayes You eagle eye, you! Thank you for catching this error. I accepted the commit suggestion because it corrects the grammar. Pending @fabicastp we can alter the casing of the phrase, too.

Reply:
In a chat with Fabiana, it was decided that we go with sentence case in our titles when possible.

Reply:
Awesome! Sounds great! @zlonko Thanks @fabicastp! 🟢